atlasdb-api/src/main/java/com/palantir/atlasdb/keyvalue/api/Namespace.java

```diff
@@ -78,7 +78,7 @@ public String getName() {
 
     @Override
     public int hashCode() {
-        return Objects.hashCode(name);
+        return 31 + Objects.hashCode(name);
```

Comment:
This produces the same result as the previous implementation, which was effectively `return Arrays.hashCode(new Object[] { name });`

Reply:
sigh. I don't know where we rely on this, of all things, but it looks like it actually makes the tests pass.